ams/ams-server/src/main/java/com/netease/arctic/ams/server/config/ServerTableProperties.java

```diff
@@ -47,5 +48,14 @@ public class ServerTableProperties {
     HIDDEN_EXPOSED.add(TableProperties.READ_DISTRIBUTION_MODE_HASH);
     HIDDEN_EXPOSED.add(TableProperties.PROTECTED_PROPERTIES.toString());
     HIDDEN_EXPOSED.add(TableProperties.TABLE_EVENT_TIME_FIELD);
+
+    HIDDEN_INTERNAL.add("meta_store_site");
```

Comment:
Could change these value with constant variables?

Reply:
Should we add a CatalogProperties class?